.github/workflows/publish-release.yml

```
@@ -88,3 +88,33 @@ jobs:
           npm-token: ${{ secrets.NPM_TOKEN }}
         env:
           SKIP_PREPACK: true
+
+  get-release-version:
+    runs-on: ubuntu-latest
+    needs: publish-npm
+    outputs:
+      RELEASE_VERSION: ${{ steps.get-release-version.outputs.RELEASE_VERSION }}
+    steps:
+      - id: get-release-version
+        shell: bash
+        run: ./scripts/get.sh ".version" "RELEASE_VERSION"
+
+  publish-release-to-gh-pages:
+    runs-on: ubuntu-latest
+    needs: get-release-version
+    name: Publish docs to `${{ needs.get-release-version.outputs.RELEASE_VERSION }}` directory of `gh-pages` branch
+    permissions:
+      contents: write
+    uses: ./.github/workflows/publish-docs.yml
+    with:
+      destination_dir: ${{ needs.get-release-version.outputs.RELEASE_VERSION }}
```

Elsewhere I see steps referenced via `steps.[step id].outputs`. e.g.:

```suggestion
    name: Publish docs to `${{ steps.get-release-version.outputs.RELEASE_VERSION }}` directory of `gh-pages` branch
    permissions:
      contents: write
    uses: ./.github/workflows/publish-docs.yml
    with:
      destination_dir: ${{ steps.get-release-version.outputs.RELEASE_VERSION }}
```

Does `needs` work as well? I've never used that before.

---

Oh I see, the example I was looking at was for reusing the output _within_ a job. Here we're using it in a separate job. Different case.